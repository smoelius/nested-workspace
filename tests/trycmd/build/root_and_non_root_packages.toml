args = ["build", "-vv", "--offline"]

[bin]
name = "cargo"

[fs]
cwd = "../../../fixtures/root_and_non_root_packages"
