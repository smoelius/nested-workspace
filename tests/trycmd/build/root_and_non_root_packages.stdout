[root_and_non_root_packages 0.1.0] [..]
